gui/src/containers/SignIn.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react'
+import { Redirect } from 'react-router'
+import PropTypes from 'prop-types'
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography'
+import { withStyles } from '@material-ui/core/styles'
+import TextField from '@material-ui/core/TextField'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { submitSignIn } from 'actions'
+
+const styles = theme => ({
+  title: {
+    marginTop: theme.spacing.unit * 5,
+    marginBottom: theme.spacing.unit * 5
+  },
+  button: {
+    margin: theme.spacing.unit * 5
+  }
+})
+
+export class SignIn extends Component {
+  state = { email: '', password: '' }
+
+  handleChange = name => event => {
+    this.setState({
+      [name]: event.target.value
+    })
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault()
+    const { email, password } = this.state
+    this.props.submitSignIn({email: email, password: password})
+  }
+
+  render () {
+    const { classes, fetching, authenticated, errors } = this.props
+    if (authenticated) {
+      return <Redirect to='/' />
+    }
+    return (
+      <form className={classes.container}
```

Comment:
I was thinking about this for #443. Do we want to have html forms as opposed to react redux form or formik?

Reply:
@NavyAdmiral I like the idea of using `formik` to keep it consistent with your PR. However, this story is blocking a bunch of users and I would prefer to push it through and do that as a refactor in another story.